assembly/pom.xml

```diff
@@ -150,6 +154,7 @@
 
   <build>
     <plugins>
+      <!--
```

Comment:
need to install and deploy the assembly module, otherwise dependencies cannot be resolved.